localstack/services/apigateway/integration.py

```diff
@@ -4,8 +4,9 @@
 from abc import ABC, abstractmethod
 from collections import defaultdict
 from http import HTTPStatus
-from typing import Dict, List, Union
+from typing import Any, Dict, List, Union
 
+from moto.apigatewayv2.exceptions import BadRequestException
```

Comment:
I think we should import this exception from `localstack.aws.api.apigateway` instead. (to decouple ourselves from `moto` over time)

If we follow the trace of how this exception bubbles up the call stack, we end up at the `ApigatewayRouter`, and our internal middleware/Gateway will be able to handle the ASF generated exception. 👍 

```suggestion
from localstack.aws.api.apigateway import BadRequestException
```